spec/array_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'array'
+describe Array do
+  it 'calculate sum of array from 0' do
+    expect([1, 2, 3].my_inject(0) { |num, sum| num + sum }).to eq 6
+    expect([1, 2, 3].inject(0) { |num, sum| num + sum }).to eq 6
```

Comment:
To keep things DRY, you could use `subject { [1, 2, 3] }` and avoid yourself some paid